### DISCUSSION TITLE
4.1.1rc1

### DISCUSSION THREAD

**Discussion by sadpandajoe** [apache/superset#30947] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 4.1.1.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/4.1.1rc1/

The Git tag for the release:
https://github.com/apache/superset/tree/4.1.1rc1

The CHANGELOG for the release:
https://github.com/apache/superset/blob/4.1.1rc1/CHANGELOG/4.1.1.md

The instructions for updating to the release:
https://github.com/apache/superset/blob/4.1.1rc1/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

If you have any usage questions or have problems when upgrading or
find any issues with enhancements included in this release, please
don't hesitate to let us know.

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/4.1.1rc1'>4.1.1rc1</a>.</em>

**Comment by scholarsmate** [comment]
+1 (non-binding)

4.1.1-RC1 Docker image seems to work fine on MacOS (Sequoia 5.1) on Docker Desktop.
